action/config/load.go

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package config
+
+import (
+	"github.com/go-vela/cli/internal/client"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/spf13/afero"
+
+	"github.com/urfave/cli/v2"
+
+	yaml "gopkg.in/yaml.v2"
+)
+
+// Load reads the config file and sets the values based off the provided configuration.
+func (c *Config) Load(ctx *cli.Context) error {
```

Comment:
This is where we add the new `Load()` function.

The purpose of this function is to load the flags from the configuration file and set them if they aren't already provided.